packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -0,0 +1,263 @@
+import React, {ComponentPropsWithoutRef, useState} from 'react'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import * as RadioGroup from '@radix-ui/react-radio-group'
+import clsx from 'clsx'
+import {Link} from 'react-router-dom'
+
+import newTemplate from '../../../../../static/images/illustrations/newTemplate.png'
+import estimatedEffortTemplate from '../../../../../static/images/illustrations/estimatedEffortTemplate.png'
+
+import {CreateNewActivityQuery} from '~/__generated__/CreateNewActivityQuery.graphql'
+import {ActivityCard} from '../ActivityCard'
+import {ActivityBadge} from '../ActivityBadge'
+
+import IconLabel from '../../IconLabel'
+import {ActivityLibraryHeader} from '../ActivityLibraryHeader'
+import NewMeetingTeamPicker from '../../NewMeetingTeamPicker'
+import sortByTier from '../../../utils/sortByTier'
+import {useHistory} from 'react-router'
+import {AddReflectTemplateMutation$data} from '../../../__generated__/AddReflectTemplateMutation.graphql'
+import useAtmosphere from '../../../hooks/useAtmosphere'
+import useMutationProps from '../../../hooks/useMutationProps'
+import AddReflectTemplateMutation from '../../../mutations/AddReflectTemplateMutation'
+import {Threshold} from '../../../types/constEnums'
+import useRouter from '../../../hooks/useRouter'
+import PrimaryButton from '../../PrimaryButton'
+
+type ActivityType = 'retrospective' | 'poker'
+
+const SUPPORTED_CUSTOM_ACTIVITIES = [
+  {
+    title: 'Process Feedback',
+    type: 'retrospective' as ActivityType,
+    includedCategories: ['retrospective', 'feedback', 'strategy'],
+    image: newTemplate,
+    isEnabled: true,
+    badges: [
+      {category: 'retrospective', theme: 'bg-grape-500 text-white', title: 'Retrospective'},
+      {category: 'feedback', theme: 'bg-jade-400 text-white', title: 'Feedback'},
+      {category: 'strategy', theme: 'bg-rose-500 text-white', title: 'Strategy'}
+    ],
+    phases: [
+      {title: 'Add', description: 'comments'},
+      {title: 'Group', description: 'comments'},
+      {title: 'Discuss', description: 'topics'},
+      {title: 'Vote', description: 'on topics'},
+      {title: 'Create', description: 'takeaway tasks'}
```

Comment:
+1 This all feels a little too clever. Maybe replace this with a `description` `React.Node`? That might make this a little more readable.

Reply:
I agree, this feels weird. Not sure I understand the solution with `React.Node` though, can you give an example? 
I also thought about creating a simple component that'd take a string as a prop, split it by ` `, make the first item bold and then join the array and display it. wdyt?

Reply:
Basically the same way a component would handle a `label` prop. So this would look like:
```
phases: (
  <>
    <div>
      <span className='font-semibold text-slate-800'>Add</span> comments
    </div>
    <div>
      <span className='font-semibold text-slate-800'>Group</span> comments
    </div>
    (etc.)
  </>
)
```
and then further down:
```
<div className='mx-5 space-y-2 text-left'>
  {activity.phasesDescription}
</div>
```

Objectively less elegant, but a bit more readable.

Reply:
ah, that. improved, and extracted the `<Bold />` component as I couldn't look at this repeated class names ;)